Doc/Core#666 - Move GlobalAppConfig attributes to CoreSection (#511)
Avaiga/taipy-doc: docs/manuals/core/config/config-checker.md

> Here is the list of the possible issues that could be returned by the checker:

Reviewer: ```suggestion
Here is the list of the possible issues that the checker could return:
```

> - An `WARNING` issue is created if the `repository_type` property is populated in the `CoreSection^` with an

Reviewer 2: ```suggestion
- A `WARNING` issue is created if the `repository_type` property is populated in the `CoreSection^` with an
```

  unsupported repository value.

> - An `ERROR` issue is created if the `storage_type` and the `scope` properties of any `DataNodeConfig^` have not

Reviewer: An `ERROR` issue is created if the `storage_type` and the `scope` properties of any `DataNodeConfig^` are not

> - An `ERROR` issue is created if the `storage_type` and the `scope` properties of any `DataNodeConfig^` have not

Reviewer: ```suggestion
- An `ERROR` issue is created if the `storage_type` and the `scope` properties of any `DataNodeConfig^` are not
```

>   been provided with a correct value.

Reviewer: provided with a correct value.

>   been provided with a correct value.

Reviewer: provided with a correct value.

>   been provided with a correct value.

Reviewer: ```suggestion
  provided with a correct value.
```
